.mbedignore

```diff
@@ -28,3 +28,4 @@ projects/adt7420_console/
 projects/ad559xr_console/
 projects/ad5933_console/
 projects/evb_discovery_firmware/
+projects/ad2s1210_iio/
```

Comment:
Could we move this above evb_discovery_firmware please? We want discovery firmware to be last in the list purposefully.